gooddata-java-model/src/main/java/com/gooddata/sdk/model/hierarchicalconfig/SourceType.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) 2004-2021, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.sdk.model.hierarchicalconfig;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.gooddata.sdk.common.GoodDataException;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Source type defines source of setting, its api uri in case it is manageable and source hierarchy of setting.
+ * For example, you can configure a whole domain and overwrite that configuration item for the project setting.
```

Comment:
I think this is not true. Project may inherit setting from domain, but if the setting is defined both in the domain and in the project, then setting from project wins (it's not overwritten from domain).

Reply:
I took it from public doc. So we need to fix it also in that. 
<img width="1792" alt="Screenshot 2021-01-14 at 14 20 35" src="https://user-images.githubusercontent.com/75017489/104923246-16f72d00-599c-11eb-9950-615de1230132.png">

Reply:
I took it from public documentation. So we need to fix it also in that. 
<img width="1599" alt="Screenshot 2021-01-18 at 14 45 03" src="https://user-images.githubusercontent.com/75017489/104923357-3beba000-599c-11eb-91e8-acf5f150ef07.png">

Reply:
Changed to: For example, the project can inherit config from the domain, but if the config is defined for both: domain and project, then config from project takes precedence.